tasks/sprite.js

```diff
@@ -0,0 +1,28 @@
+const gulp = require('gulp');
+const config = require('./../config');
+
+gulp.task('convertIconsToSprites', function() {
```

Comment:
This extra task is not necessary if everything is handled inside a single task - see comment below in the copy task